epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -86,10 +104,12 @@ public ClassToGenerateInfo(TypeElement originalClassName, ClassName generatedCla
       };
 
   public void addAttribute(AttributeInfo attributeInfo) {
+    removeMethodIfDuplicatedBySetter(Collections.singletonList(attributeInfo));
```

Comment:
might as well do `addAttributes(Collections.singletonList(attributeInfo))` to remove this duplication